src/types.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Represents the proxy object, which wraps a target object. As a result, it
+ * allows for accessing and setting all of the properties that the target object
+ * supports, but also supports an extra method to switch the target.
+ *
+ * @member setTarget - Allows for switching the target.
```

Comment:
This isn't the right tag to use; the proper tag here is `@property`. However, TypeDoc doesn't seem to parse `@property` correctly (it completely ignores the name of the property). `@member` at least makes this whole line show up.

Reply:
Interesting; in the past I've had success with adding a comment block above the property itself. Never tried `@member`.

Reply:
Yeah, I think that approach works for interfaces, but not for type aliases. TypeDoc seems to handle those differently.

Reply:
Ahh I see, interesting